package.json

```diff
@@ -62,5 +62,11 @@
     "release:notification": "zx scripts/create-slack-notification.mjs",
     "slack": "zx scripts/slack-release-notification.mjs --quiet",
     "prepare": "husky install"
+  },
+  "devDependencies": {
```

Comment:
wie only have dependencies in the root, if they are in the root can we remove storybook from the `config` too? and remove it from the workspaces.